microshift_getting_started/microshift-architecture.adoc

```diff
@@ -43,7 +43,7 @@ The following operational differences from {oke} can help you understand where {
 .{product-title} differences from {oke}.
 image::311_RHDevice_Edge_Overview_0223_2.png[<{microshift-short} is tasked with only the Kubernetes cluster capabilities of networking, ingress, storage, helm, with the additional Kubernetes functions of orchestration and security, as the following diagram illustrates.>]
 
-Figure 2 shows that {oke} has the same cluster capabilities as {product-title}, and adds the following:
+Figure 2.2 shows that {oke} has the same cluster capabilities as {product-title}, and adds the following:
```

Comment:
Should the actual image be titled 2.2? On the same page, I see you're referencing "Figure 2.2," however, I do not see a "Figure 2.2". I see a "Figure 2". 

We need a noun after "following". For example: 

```suggestion
Figure 2.2 shows that {oke} has the same cluster capabilities as {product-title}, and adds the following information:
```

Reply:
Actually the preview shows Figure 2 but the actual customer portal shows Figure 2.2. So I have removed the numbering and added the title to resolve this issue.